tutordistro/patches/cms-env

```diff
@@ -1,2 +1,2 @@
 USE_EOX_TENANT: True
-DEFAULT_SITE_THEME: "bragi"
+DEFAULT_SITE_THEME: "{{ DISTRO_THEMES_NAME[0] }}"
```

Comment:
I have been thinking on this, what happen if a person have many themes and wants to change to anyone that is not the first? The person should to change the order in DISTRO_THEMES_NAME?

Maybe we can set a new settings named "DISTRO_DEFAULT_SITE_THEME" and change this for something like:

```suggestion
DEFAULT_SITE_THEME: "{{ DISTRO_DEFAULT_SITE_THEME | default(DISTRO_THEMES_NAME[0] ) }}"
```
What do you think?

Reply:
If you agree with this, please, test my suggestion with the variables in config.yml and without the variable, I don't know if it will works.

Reply:
@Alec4r I do agree. Is just that I'm not the most proficient person in this type of scripts nor their syntax. Therefore, is quite hard for me to propose complex solutions. Please let me know if how's written works accurately so I can introduce them into the new solution and try them to update this PR.

Reply:
@danielsgz you could use [This](https://support.sendwithus.com/jinja/default/#:~:text=Jinja%20has%20a%20built%20in,when%20one%20isn%27t%20provided.&text=If%20the%20value%20isn%27t,value%20is%20defined%20but%20blank.) as guide.

Reply:
@Alec4r just changed and tried it. Working just fine.